dashboard/src/features/accessTokens/components/List.jsx

```diff
@@ -1,17 +1,33 @@
+import React from 'react'
 import { BaseList, TableList } from 'features/shared/components'
 import Item from  './ListItem'
 import actions from 'actions'
 
 const clientType = 'client_access_token'
 const networkType = 'network_access_token'
 
-const stateToProps = (type) => (state, ownProps) =>
+const emptyContentClient = <div className="emptyContainer">
```

Comment:
(I realize this branch is pretty far along so I am not married to this - just want to keep it in mind)